web/templates/dropdown_widget_with_label.hbs

```diff
@@ -3,5 +3,7 @@
         {{#if help_link}}{{> help_link_widget link=help_link }}{{/if}}
     </label>
     <span class="prop-element hide" id="id_{{widget_name}}" data-setting-widget-type="dropdown-list-widget" {{#if value_type}}data-setting-value-type="{{value_type}}"{{/if}}></span>
-    {{> dropdown_widget}}
+    <div class="inline-block dropdown_widget_with_label_wrapper">
+        {{> dropdown_widget}}
+    </div>
```

Comment:
I made this wrapper here, since this seemed to be the only approach so as to show the tooltip only on the dropdown button.